app/static/src/services/staffService.js

```diff
@@ -0,0 +1,24 @@
+import ServerConnector from "./serverConnector";
+let instance = null;
+export default class StaffService extends ServerConnector{
+    constructor(){
+        super();
+        this.path = "api/staff"
+    }
+    static getInstance() {
+        if (!instance) {
+            instance = new StaffService();
```

Comment:
and here write this
instance = new StaffService("api/staff");

And maybe it is better to  rename it to EmployeesService